lib/node/NodeWatchFileSystem.js

```diff
@@ -57,7 +57,8 @@ class NodeWatchFileSystem {
 				changes.filter(file => dirs.includes(file)).sort(),
 				changes.filter(file => missing.includes(file)).sort(),
 				times,
-				times
+				times,
+				removals
```

Comment:
filter it by `files`, `dirs` or `missing`, sort it, convert it to a `Set`

Reply:
👍

Reply:
Updated @sokra 👍